backend/setup.js

```diff
@@ -1,7 +1,19 @@
 const downloadOpenMINDS = require('./internal/update/download_openminds.js');
+//const fetchCoreSchemaInstances = require('./internal/update/fetchCoreSchemaInstances'); // Todo?
+//const fetchControlledTerms = require('./internal/update/fetchControlledTerms'); // Todo?
+const resolveSpeciesInStrain = require('./internal/update/resolveSpeciesInStrain'); // Todo?
 
+const getRequestOptions = require('./internal/update/getRequestOptions'); // Todo?
+
+
+resolveSpeciesInStrain
```

Comment:
why is this here

Reply:
The setup method is incomplete, and should note have been part of this PR. Please accept, as I don't have time to clean up the git history now.

Reply:
I more mean the line resolveSpeciesStrain seems to just be there without doing anything. Cant we just delete it?